### PR TITLE
Jpk/invoice pay

### PR DESCRIPTION
**Pull request by jpknoll**

### CODE REVIEW

File: src/Payments.Mvc/Views/Shared/_PaymentsLayout.cshtml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <base href="~/" />
+    <environment names="Development">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.css" />
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.css" />
+    </environment>
+    <environment names="Staging,Production">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.css" />
+        <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
+    </environment>
```
Comment (srkirkland) [inline src/Payments.Mvc/Views/Shared/_PaymentsLayout.cshtml]: @jpknoll maybe put this stuff into common partials, I know several layout files will be using these + the js ones, and I know they will be updated soon, bootstrap for example is already on version 4.1

File: src/Payments.Mvc/Views/Payments/Pay.cshtml [hunk 2]
```diff
@@ -1,19 +1,92 @@
...
+    ViewBag.Title = $"Pay Invoice {Model.Id}";
+    Layout = "_PaymentsLayout";
 }
 
-<h2>title</h2>
-<form action="@ViewBag.CyberSourceUrl" method="post" autocomplete="off" style="margin-right: 3px">
-    @foreach (var pair in Model.PaymentDictionary)
+<div class="container">
+    <h1>Invoice from UCDavis Arboretum</h1>
+    <p>Billed to jpknoll@ucdavis.edu</p>
```
Comment (srkirkland) [inline src/Payments.Mvc/Views/Payments/Pay.cshtml]: Temporary i'm assuming

Comment (jpknoll) [reply src/Payments.Mvc/Views/Payments/Pay.cshtml]: Yeah, just trying to get a baseline for Cal to work off of.

File: src/Payments.Mvc/Views/Shared/_PaymentsLayout.cshtml (hunk 1 above)
Comment (jpknoll) [reply src/Payments.Mvc/Views/Shared/_PaymentsLayout.cshtml]: I'm fine with making a bootstrap/fontawesome partial.

**Comment by jpknoll** [comment]
Agreed on the commonality of the two containers. They differ on their save methods and treatment of multiple customers. I'll continue to abstract out various inputs and components.

**Pull request merged**